site/en/docs/extensions/mv3/declare_permissions/index.md

```diff
@@ -129,348 +122,253 @@ If your extension needs to run on `file://` URLs or needs to operate in incognit
+
+`"declarativeNetRequestWithHostAccess"`
+: Gives extension access to the <a href="/docs/extensions/reference/declarativeNetRequest/">chrome.declarativeNetRequest</a> API. Also requires host permissions to the request URL and initiator to act on a request.
+
+`"declarativeNetRequestFeedback"`
+: Gives access to events and methods within the <a
+          href="/docs/extensions/reference/declarativeNetRequest/">chrome.declarativeNetRequest</a> API, which returns information on declarative rules matched.
+
+<!-- No corresponding reference entry
+    `"displaySource"`
+    : Gives access to the <a href="/docs/extensions/reference/displaySource/">chrome.displaySource</a> API.
+
+    `"dns"`
+    : Gives access to the <a href="/docs/extensions/reference/dns/">chrome.dns</a> API.
+-->
+<!--Do these still need to be commented out?-->
+
+`"desktopCapture"`
+: Gives access to the <a href="/docs/extensions/reference/desktopCapture/">chrome.desktopCapture</a> API.
+
+`"documentScan"`
+: Gives access to the <a href="/docs/extensions/reference/documentScan/">chrome.documentScan</a> API.
+
+`"downloads"`
+: Gives access to the <a href="/docs/extensions/reference/downloads/">chrome.downloads</a> API.
+
+`"downloads.open"`
+: Allows the use of <a href="/docs/extensions/reference/downloads/#method-open">chrome.downloads.open()</a>.
+
+`"downloads.ui"`
+: Allows the use of <a href="/docs/extensions/reference/downloads/#method-setUiOptions">chrome.downloads.setUiOptions()</a>.
+
+`"enterprise.deviceAttributes"`
+: Gives access to the <a
+          href="/docs/extensions/reference/enterprise_deviceAttributes/">chrome.enterprise.deviceAttributes</a> API.
+
+`"enterprise.hardwarePlatform"`
+: Gives access to the <a
+          href="/docs/extensions/reference/enterprise_hardwarePlatform/">chrome.enterprise.hardwarePlatform</a> API.
+
+`"enterprise.networkingAttributes"`
+: Gives access to the <a        href="/docs/extensions/reference/enterprise_networkingAttributes/">chrome.enterprise.networkingAttributes</a> API.
+
+`"enterprise.platformKeys"`
+: Gives access to the <a href="/docs/extensions/reference/enterprise_platformKeys/">chrome.enterprise.platformKeys</a> API.
+
+`"experimental"`
+: Allows the use of <a href="/docs/extensions/reference/#experimental_apis/">chrome.experimental.* APIs</a>.
+
+`"fileBrowserHandler"`
+: Gives access to the <a href="/docs/extensions/reference/fileBrowserHandler/">chrome.fileBrowserHandler</a> API.
+
+`"fileSystemProvider"`
+: Gives access to the <a href="/docs/extensions/reference/fileSystemProvider/">chrome.fileSystemProvider</a> API.
+
+`"fontSettings"`
+: Gives access to the <a href="/docs/extensions/reference/fontSettings/">chrome.fontSettings</a> API.
+
+`"gcm"`
+: Gives access to the <a href="/docs/extensions/reference/gcm/">chrome.gcm</a> API.
+
+`"geolocation"`
+:Allows the extension to use the <a
+          href="https://dev.w3.org/geo/api/spec-source.html">geolocation API</a> without prompting the user for permission.
```

Comment:
```suggestion
          href="https://developer.mozilla.org/docs/Web/API/Geolocation_API">geolocation API</a> without prompting the user for permission.
```

Reply:
If you find any references to w3c or wecg specs, please look for them on [MDN](https://developer.mozilla.org/) and use that link instead. The links you copy will contain a country code ('en-US/'). Please remove that from the URL.